cmd/snap-update-ns/utils_test.go

```diff
@@ -315,6 +315,23 @@ func (s *utilsSuite) TestSecureMkfileAllLevel3(c *C) {
 	})
 }
 
+// Ensure that we can detect read only filesystems.
+func (s *utilsSuite) TestSecureMkfileAllROFS(c *C) {
+	s.sys.InsertFault(`mkdirat 3 "rofs" 0755`, syscall.EEXIST) // just realistic
+	s.sys.InsertFault(`openat 4 "path" O_NOFOLLOW|O_CLOEXEC|O_RDWR|O_CREAT|O_EXCL 0755`, syscall.EROFS)
+	err := update.SecureMkfileAll("/rofs/path", 0755, 123, 456)
+	c.Check(err, ErrorMatches, `cannot operate on read-only filesystem at /rofs`)
+	// c.Assert(err.(*update.ReadOnlyFsError).Path, Equals, "/rofs")
```

Comment:
Did you mean to leave this commented out?

Reply:
No, good catch. Re-enabled now.